From 4db707c6f47c40a278c3b58070056240b9dcfb07 Mon Sep 17 00:00:00 2001
From: Mauro Carvalho Chehab <mchehab@s-opensource.com>
Date: Wed, 7 Mar 2018 04:58:49 -0500
Subject: [PATCH 288/437] media: cxd2880: Makefile: remove an include

It is not needed anymore to include the dvb-core directory,
as all the public headers that used to be there was moved
to include/media.

Signed-off-by: Mauro Carvalho Chehab <mchehab@s-opensource.com>
---
 drivers/media/dvb-frontends/cxd2880/Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/media/dvb-frontends/cxd2880/Makefile b/drivers/media/dvb-frontends/cxd2880/Makefile
index 65a5d37f28cc..c6baa4caba19 100644
--- a/drivers/media/dvb-frontends/cxd2880/Makefile
+++ b/drivers/media/dvb-frontends/cxd2880/Makefile
@@ -15,5 +15,4 @@ cxd2880-objs := cxd2880_common.o \
 
 obj-$(CONFIG_DVB_CXD2880) += cxd2880.o
 
-ccflags-y += -Idrivers/media/dvb-core
 ccflags-y += -Idrivers/media/dvb-frontends
-- 
2.17.0

